Tests/Functional/DataProcessing/AddWallsProcessorTest.php

```diff
@@ -96,19 +81,20 @@ public function processWithValidTypoScriptConditionWillNotModifyProcessedData():
             ],
         ];
 
-        $this->contentObjectRendererProphecy
-            ->checkIf($processorConfiguration['if.'])
-            ->shouldBeCalled()
+        $this->contentObjectRendererMock
+            ->method('checkIf')
+            ->with($processorConfiguration['if.'])
```

Comment:
The equivalent for shouldBeCalled is missing.

```suggestion
        $this->contentObjectRendererMock
            ->expects($this->atLeastOnce())
            ->method('checkIf')
            ->with($processorConfiguration['if.'])
```